Expecto/Logging.fs

```diff
@@ -12,10 +12,12 @@
 ///
 /// Changes:
 ///     Changed namespace to Expecto.Logging and file name to Logging.fs - Anthony Lloyd - 11 Jun 2018
+///     Add IFlushable - Anthony Lloyd - 19 Jun 2018
```

Comment:
Why keep a changelog at the top? Feels a bit year 2000.

Reply:
I read that it was a good thing to do when making changes to Apache code. To help when back porting the code. Could be got from git but it contains a bit more specific info.